doc/classes/ProceduralSky.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ProceduralSky" inherits="Sky" category="Core" version="3.1">
 	<brief_description>
+		Type of [Sky] that is generated procedurally based on user input parameters.
 	</brief_description>
 	<description>
+		The ProceduralSky provides a means for users to create an effective background quickly for prototyping or to create a background for a custom scene. It has three main components: the sun, the sky, and the ground. The sky and ground are very similar, they are defined by a color at the horizon, another color, and finally an easing curve to interpolate between these two colors. Similarly the sun is described by a position in the sky, a color, and an easing curve. However, the sun also defines a minimum and maximum angle, these two values define at what distance the easing curve begins and ends from the sun, and thus end up defining the size of the sun in the sky.
```

Comment:
IMO, shorten first two sentences to "ProceduralSky provides a way to create an effective background quickly by defining procedural parameters for the sun, the sky and the ground."

Reply:
I like it. Thanks for the feedback!